api/src/main/java/io/minio/MinioClient.java

```diff
@@ -4840,14 +4840,47 @@ public String getBucketLifeCycle(GetBucketLifeCycleArgs args)
    * @throws NoSuchAlgorithmException thrown to indicate missing of MD5 or SHA-256 digest library.
    * @throws XmlParserException thrown to indicate XML parsing error.
    */
+  @Deprecated
```

Comment:
Good to add deprecation comment also with link to new method

Reply:
Done